src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdk.diff

```diff
@@ -428,108 +428,10 @@ index ------------
 -./sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Drawing.Common.dll
  ./sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Security.Cryptography.Pkcs.dll
 -./sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Security.Cryptography.ProtectedData.dll
- ./sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Windows.Extensions.dll
+-./sdk/x.y.z/Containers/tasks/netx.y/runtimes/win/lib/netx.y/System.Windows.Extensions.dll
  ./sdk/x.y.z/Containers/tasks/netx.y/System.CommandLine.dll
  ./sdk/x.y.z/Containers/tasks/netx.y/System.Configuration.ConfigurationManager.dll
-@@ ------------ @@
- ./sdk/x.y.z/cs/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.resources.dll
- ./sdk/x.y.z/cs/Microsoft.TemplateEngine.Utils.resources.dll
- ./sdk/x.y.z/cs/Microsoft.TemplateSearch.Common.resources.dll
-+./sdk/x.y.z/cs/Microsoft.TestPlatform.Build.resources.dll
```

Comment:
So what's the story for this file? This doesn't seem addressed by the patch.

Reply:
Good catch - this is in a separate nupkg: `Microsoft.TestPlatform.Build`

Working on a fix.

Reply:
Something with SDK composition is incorrect - this localized binary was already included in the relevant nupkg.

Reply:
Argh - this is funny - this is just a diff showing what's **not** in baseline anymore. No issue here.

Reply:
Oh, gah. META-DIFFS!